trikot-kword/kword/src/androidMain/kotlin/com/mirego/trikot/kword/android/PreviewI18N.kt

```diff
@@ -0,0 +1,49 @@
+package com.mirego.trikot.kword.android
+
+import com.mirego.trikot.kword.DefaultI18N
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import java.io.File
+
+/**
+ * When using the Jetpack Compose previewer in Android Studio, it may be useful to have the actual translations. Since it's
+ * running in a different context than the app, the default Android implementation of I18N is not working.
+ *
+ * This implementation loads a translation file using an absolute path to the translations file in the developer environment.
+ *
+ * Here is a suggestion on how to setup your project to get this path automatically (without it being mandatory).
+
+ * First, add an entry to the non version-controlled local.properties:
+ * ```
+ * translation_file_path=/absolute/path/to/translation/file.json
+ * ```
+ *
+ * Then, add this snippet to your build.gradle file (example in Kotlin)
+ * ```
+ * val properties = gradleLocalProperties(rootDir)
+ * val translationFilePath: String = properties.getProperty("translation_file_path", "")
```

Comment:
I feel we could do better than that and resolve the absolute path using gradle project root or root dir?
```
project.rootDir
project.projectDir
```

Reply:
Hmm, maybe. I've tried many different ways to get this information at runtime in the Previewer context, without success. But you're right, at the gradle level it would be easier to get it.

Reply:
@npresseault This was a really good idea, thanks! 👍